### ISSUE TITLE
Δ for sul-embed consuming IIIF v3 for all viewers

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/purl#164]
- [ ] file sizes
- [ ] resource labels
- [ ] embargo information
- [ ] memento links
- [ ] use and reproduction / copyright statements
- [ ] bounding boxes
- [ ] contentMetadata type information

**Comment by mejackreed** [comment]
It would be great if the bounding boxes could be serialized as GeoJSON.

**Comment by jcoyne** [comment]
To be clear this is a list of data that sul-embed consumes which is not provided by IIIF v3, right?

**Comment by cbeer** [comment]
Indeed. And some are probably things we can do better about on our own, but some probably need to turn into IIIF feature requests.
